app/javascript/components/Toolbar/index.js

```diff
@@ -39,12 +39,16 @@ const Toolbar = ({ label, view, views, onNavigate, onViewChange, filters, office
         </button>
       </div>
       <div className={s.filterBar}>
-        <div className={s.filterDropdowns}>
-          <ShowFilter value={filters.showFilter.value} onChange={filters.showFilter.onChange} />
-          <EventFilter value={filters.eventFilter.value} onChange={filters.eventFilter.onChange} />
-          <OfficeFilter value={filters.officeFilter.value} onChange={filters.officeFilter.onChange} offices={offices} />
+        <div className="u-display-inline-flex u-mb-xxs">
```

Comment:
I wonder if it's bad to be mixing the use of both the spacing utility classNames with styled components. I got kind of confused where these class names came from (but then I googled ha).

Is it possible to extract the spacing utilities from https://garden.zendesk.com/css-components/utilities/spacing/ and access them via styled components and use `styled.div` or something along that line?

Maybe not in this PR, but just a thought for the future 🤔

Reply:
I also prefer styled components over the class names. I can access the spacing values through the garden css variables. I will do that instead :)

Reply:
@samzx what are your thought on using something like this? https://styled-system.com
I have used it before and really like it. When done right, you end up with far less components.

Reply:
Looks like an interesting concept 😮 I'll have to have a try to see what it is